backend/package.json

```diff
@@ -6,7 +6,7 @@
   "main": "server.js",
   "scripts": {
     "start": "node server.js",
-    "dev": "./node_modules/nodemon/bin/nodemon.js -L server.js",
```

Comment:
This was added to be compatible with docker-compose on MacOS. Please revert this change